nms/latest/src/main/java/com/nisovin/magicspells/volatilecode/latest/VolatileGlowManagerLatest.java

```diff
@@ -43,6 +44,9 @@
 
 public class VolatileGlowManagerLatest extends PacketBasedGlowManager<Packet<?>, ClientboundSetEntityDataPacket, ClientboundSetPlayerTeamPacket> {
 
+	public static final StringRepresentable.EnumCodec<Visibility> VISIBILITY_CODEC = StringRepresentable.fromEnum(Visibility::values);
+	public static final StringRepresentable.EnumCodec<CollisionRule> COLLISION_RULE_CODEC = StringRepresentable.fromEnum(CollisionRule::values);
+
```

Comment:
Storing these to fields seemed like the best way because the vanilla enums don't expose `CODEC` as `EnumCodec` for us to use `EnumCodec#byName`, so we can recreate it instead with the right field type.

![image](https://github.com/user-attachments/assets/420adf30-a0cb-472e-b049-0fcf1e042428)

Reply:
It's not really necessary to store them as fields? Also, perhaps use `StringRepresentable#createNameLookup` instead of creating a codec?